guides/doc-Content_Management_Guide/topics/Containers_interacting_with_registry.adoc

```diff
@@ -0,0 +1,32 @@
+[[using_container_registries]]
+= Using Katello Container Registries
+
+Podman and Docker can be used to fetch content from Katello container registries.
+
+.Container Registries on Smart Proxies
+
+On Smart Proxies with content, the https://github.com/Katello/smart_proxy_container_gateway[Container Gateway] smart proxy plugin acts as the container registry.  It caches authentication information from Katello and proxies incoming requests to Pulp.  The Container Gateway comes default on Smart Proxies with content.
+
+.Procedure
+
+Logging in to the container registry:
+[options="nowrap", subs="+quotes,attributes"]
+----
+podman login {foreman-example-com}
+----
+
+Listing container images:
+[options="nowrap", subs="+quotes,attributes"]
+----
+podman search {foreman-example-com}/
+----
+
+Pulling container images:
+[options="nowrap", subs="+quotes,attributes"]
+----
+podman pull {foreman-example-com}/my-image:<optional_tag>
+----
+
+.Limitations
+
+Please note that, at first release with Katello 4.0, the Container Gateway will not yet support pulling container images that require authentication.  Until it does, ensure that *Unauthenticated Pull* is checked for all Lifecycle Enviroments that have container repositories that are expected to be served by Smart Proxies.  This is planned to be resolved by Katello 4.1.
```

Comment:
```suggestion
With the Katello 4.0 release, the Container Gateway does not support pulling container images that require authentication.  Until it does, ensure that *Unauthenticated Pull* is checked for all Lifecycle Environments that have container repositories that are expected to be served by {SmartProxies}.  This is planned to be resolved by Katello 4.1.
```

I would add the link to the redmine issue and say "Until x is resolved " rather than make any promises. Can you please add an ifdef for katello around this bit for fear it would get into Satellite docs in time :)